subprojects/internal-testing/src/main/groovy/org/gradle/test/fixtures/concurrent/TestExecutor.groovy

```diff
@@ -67,7 +70,7 @@ class TestExecutor implements Executor {
                     logger.log "finished"
                     lock.lock()
                     try {
-                        threads.remove(Thread.currentThread())
+                        TestExecutor.this.threads.remove(currentThread())
```

Comment:
Why change this line? Why not:
```
threads.remove(currentThread())
```

Reply:
IntelliJ was confused about what `this` is in the context, and was producing a warning. I opted to be more explicit and make the warning go away.